cmd/codescanning.go

```diff
@@ -13,21 +15,26 @@ import (
 
 func init() {
 	codeScanningCmd.PersistentFlags().StringVarP(&Organization, "organization", "o", "", "specify Organisation to implement code scanning")
-	codeScanningCmd.MarkPersistentFlagRequired("organization")
-	codeScanningCmd.PersistentFlags().StringVarP(&WorkflowFile, "workflow-file", "f", "", "specify the path to the code scanning workflow file")
-	codeScanningCmd.MarkPersistentFlagRequired("workflow-file")
-	codeScanningCmd.PersistentFlags().StringVarP(&LogFile, "log-file", "l", "", "specify the path where the log file will be saved")
-	codeScanningCmd.MarkPersistentFlagRequired("log-file")
-
+	codeScanningCmd.PersistentFlags().StringVarP(&WorkflowFile, "workflow", "w", "", "specify the path to the code scanning workflow file")
+	codeScanningCmd.MarkPersistentFlagRequired("workflow")
+	codeScanningCmd.PersistentFlags().StringVarP(&LogFile, "log", "l", "code-scanning.out", "specify the path where the log file will be saved")
+	codeScanningCmd.PersistentFlags().StringVarP(&CsvFile, "csv", "c", "", "specify the location of csv file")
+	// MarkFlagsOneRequired is only available in cobra v1.8.0 that still isn't released yet (https://github.com/spf13/cobra/issues/1936#issuecomment-1669126066)
+	// codeScanningCmd.MarkFlagsOneRequired("csv", "organization")
+	codeScanningCmd.MarkFlagsMutuallyExclusive("csv", "organization")
 }
 
 var codeScanningCmd = &cobra.Command{
-	Use:   "code-scanning-enable-all",
+	Use:   "code-scanning",
 	Short: "Add workflow files to enable code scanning",
 	Long:  "Creates branch `code-scanning-automated` on each repo in organisation and checks in workflow file defined in `--workflow` flag",
 	Run: func(cmd *cobra.Command, args []string) {
 
 		//set up logging
+		if len(LogFile) <= 0 {
+			LogFile = "code-scanning.out"
+		}
```

Comment:
Should we print the creation of the log file out to the user so that they are aware that it exists in a certain location?

Reply:
Great idea!